src/KawaBot.Server/OAuthController.fs

```diff
@@ -0,0 +1,97 @@
+﻿namespace KawaBot.Server
+
+open System.IO
+open Http
+open System.Net.Http
+open System.Net
+open Newtonsoft.Json
+open Environment
+open System.Collections.Generic
+open Newtonsoft.Json.Linq
+open System.Text
+open System.Threading.Tasks
+open System
+
+[<JsonObject>]
```

Comment:
Lets create `module OAuthController =` for this file. As it's done in the `HealthController`. What do you think?